index.bs

```diff
@@ -939,6 +978,8 @@ To <dfn>process an attribution source</dfn> given an [=attribution source=] |sou
         1. [=set/Append=] |fakeReport| to the [=event-level report cache=].
     1. If |source|'s [=attribution source/randomized response=] is not [=set/is empty|empty=],
         then set |source|'s [=attribution source/event-level attributable=] value to false.
+    1. Run [=obtain and deliver a debug report on source registration=] with "`source-noised`" and |source|.
+    1. Return.
```

Comment:
Just making sure this new `return` is intentional.

Reply:
Good catch, it's not.